src/components/layout.scss

```diff
@@ -1206,7 +1206,12 @@ figcaption {
 }
 
 body {
-  transition: background 300ms ease;
+  opacity: 0;
```

Comment:
Hi @kevin-chassagne,
opacity:0 on body seam has issue with lighthouse, i tried with 0.01 and it's ok. I hope that it doesn't affect SEO performance